src/meta/src/barrier/notifier.rs

```diff
@@ -63,31 +60,10 @@ impl Notifier {
         }
     }
 
-    /// Notify when we have finished a barrier from all actors. This function consumes `self`.
```

Comment:
btw, we can change to consume the ownership of notifier for method `notify_collected` because it is expected there won't be call on the notifier after calling it.